ethereum/contracts/BasicInboundChannel.sol

```diff
@@ -7,11 +7,14 @@ contract BasicInboundChannel {
     uint256 public constant MAX_GAS_PER_MESSAGE = 100000;
     uint256 public constant GAS_BUFFER = 60000;
 
+    uint8 public constant sourceChannelID = 0;
```

Comment:
maybe put in constructor and make immutable?

Reply:
I thought about that but don't think its necessary. These channel IDs will never change and so leaving them as static constants should be fine.

Reply:
sure, good idea :+1: